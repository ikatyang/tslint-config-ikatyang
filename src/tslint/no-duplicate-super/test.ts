// good

class good1 extends Error {
  constructor() {
    super();
  }
}

// bad

class bad1 extends Error {
  constructor() {
    super();
    super();
  }
}
