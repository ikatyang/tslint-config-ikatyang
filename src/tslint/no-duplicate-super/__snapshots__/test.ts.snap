// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
// good

class good1 extends Error {
  constructor() {
    super();
  }
}

// bad

class bad1 extends Error {
  constructor() {
    super();
    ~~~~~~~~
    super();
~~~~~~~~~~~  [Multiple calls to 'super()' found. It must be called only once.]
  }
}

`;
